ceph-windows-test/build/run_tests

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o pipefail
+
+CEPH_WIN_BIN_DISTRO=${CEPH_WIN_BIN_DISTRO:-"windows"}
+CEPH_WIN_BIN_DISTRO_VERSION=${CEPH_WIN_BIN_DISTRO_VERSION:-"1809"}
+CEPH_WIN_BIN_SHA1=${CEPH_WIN_BIN_SHA1:-"c7f2056f7415c5d26b0503e7c13a89e6b4856341"}
+
+VM_IMAGE_URL=${VM_IMAGE_URL:-"https://filedump.ceph.com/windows/ceph-win-ltsc2019-ci-image.qcow2"}
+VM_IMAGES_DIR=${VM_IMAGES_DIR:-"$HOME/vm_images"}
```

Comment:
> I could simply download the VM image to the $WORKSPACE, and when a new job triggers, I think it re-uses the same $WORKSPACE as the previous job, right ?

Without `wipe-workspace: true`, yes it will reuse the same directory.

Reply:
Caching is worth if if it takes a long time to download, but it does use up space on the build machine, which can be an issue.  I guess we can change our minds later if it becomes an issue.

Reply:
@dmick, the image is downloaded super fast from the current location at:
https://filedump.ceph.com/windows/ceph-win-ltsc2019-ci-image.qcow2

As far as I understood, the web server is somewhere close to Jenkins infra, so that's why.

If we could always have the VM image hosted into a web server that allows the Jenkins agents fast download speeds, I would remove the caching logic (because it even simplifies the bash script).

The VM qcow2 is not even shared among jobs, because we want to use a clean environment on every job run. So, we copy it from the cached location, on each job run. At the moment, the copy operation takes as much as downloading it directly.

Therefore, if the download speed is always fast, there's no value in caching the image.

Reply:
@dmick @djgalloway

Given the above conversation, let me know what are you thoughts about removing the caching logic to simplify the bash script.

Reply:
If it works now, leave it; we can change it later if space is a problem or if we find a caching error.

Reply:
> If it works now, leave it; we can change it later if space is a problem or if we find a caching error.

sounds good